geth/utils/validation.py

```diff
@@ -22,28 +22,20 @@ class GethKwargs(BaseModel):
         autodag (bool): generate a DAG (pre-merge only)
         allowinsecure (bool): Allow insecure accounts.
         data_dir (str): The directory to store the chain data.
-        etherbase (str): The account to send mining rewards to.
```

Comment:
I'm going to list all the keys I did not see put back in here

- `etherbase`
- `targetgaslimit`
- `extradata`
- `genesis`
- `gasprice`
- `metrics`
- `pprof`
- `vmodule`

I'm kind of curious where these came from to begin with (and we missed, my fault included). They don't seem to be used anywhere, pre-type-hinting. I think everything else lines up with `construct_popen_command` though so I think this lgtm?

I see @kclowes had a similar comment so I'll wait until that's all resolved.

Reply:
co-pilot filled 'em in when I first created the GethKwargs model, just cleaning up/paring back to what we actually use.